distribution/docs/src/main/resources/content/_reference/_appReferences/mg-spatial.adoc

```diff
@@ -47,6 +47,45 @@ index again.
 
 |===
 
+=== Standalone Solr Offline Gazetteer Feature
+
+The Standalone Solr Offline Gazetteer is a reflection of the gazetteer data stored
+in the catalog but separated from other data. This allows the core to be
```

Comment:
✏️ `This allows the suggester to be separately rebuilt much quicker than for the entire catalog collection.`